Description:  Edit configure.ac to select armv6 when using "arm-linux-gnueabihf"
Author: Peter Michael Green <plugwash@raspbian.org>

diff --git a/configure.ac b/configure.ac
index a8e42222..36d454f6 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1375,7 +1375,7 @@ AS_CASE([$host],
   [armv5*-*-linux-*eabi],
     [arch=arm; model=armv5; system=linux],
   [arm*-*-linux-*eabihf],
-    [arch=arm; system=linux],
+    [arch=arm; model=armv6; system=linux],
   [arm*-*-linux-*eabi],
     [arch=arm; system=linux],
   [arm*-*-openbsd*],
